apps/admin_app/lib/admin_app_web/templates/layout/sidebar.html.eex

```diff
@@ -1,64 +1,102 @@
-      <div class="dropdown-menu p-0 " aria-labelledby="dropdownMenuButton">
-         <a class="dropdown-item" href="#">Back to store</a>
-         <a class="dropdown-item" href="#">Account</a>
-          <%= link("LogOut", to: session_path(@conn, :new), method: :delete, class: "dropdown-item")%>
+      <div class="dropdown-menu shopdpdown" aria-labelledby="dropdownMenuButton">
+        <a class="dropdown-item" href="#">Action</a>
+        <a class="dropdown-item" href="#">Another action</a>
+        <a class="dropdown-item" href="#">Something else here</a>
       </div>
-   </li>
-</ul>
+    </div>
+  </li>
+  <li>
+    <a class="nav-link" href="/orders">
+    <i class="fa fa-clipboard-list" aria-hidden="true"></i> Orders
+    </a>
+  </li>
+  <li>
+    <a class="nav-link" href="#">
+    <i class="fa fa-exchange-alt" aria-hidden="true"></i> Returns
+    </a>
+  </li>
+  <li>
+    <a class="nav-link" href="#" data-toggle="collapse" data-target="#drilldown-2">
+    <i class="fa fa-shapes" aria-hidden="true"></i> Products
+    </a>
+    <ul id="drilldown-2" class="nav nav-pills nav-stacked collapse pl-3">
+      <li>
+        <a class="nav-link" href="/products">Product Catalogue</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/option_types">Option Types</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/taxonomy">Product Taxonomy</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/properties">Properties</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/variation_themes">Variation Themes</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/prototypes">Prototypes</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/product_brands">Product Brands</a>
+      </li>
+    </ul>
+  </li>
+  <li>
+    <a class="nav-link" href="#">
+    <i class="fa fa-chart-bar" aria-hidden="true"></i> Reports
+    </a>
+  </li>
+  <li>
+    <a class="nav-link" href="/users">
+    <i class="fa fa-users" aria-hidden="true"></i> Users
+    </a>
+  </li>
+  <li>
+    <a class="nav-link" href="#" data-toggle="collapse" data-target="#drilldown-1">
+    <i class="fa fa-sliders-h" aria-hidden="true"></i> Configurations
+    </a>
+    <ul id="drilldown-1" class="nav nav-pills nav-stacked collapse pl-3">
+      <li>
+        <a class="nav-link" href="#">Stock Items</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/stock_locations">Stock Locations</a>
+      </li>
+      <li>
+        <a class="nav-link" href="#">Stock Movements</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/roles"> Roles</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/permissions">Permission</a>
+      </li>
+      <li>
+        <a class="nav-link" href="/payment_methods">Payment Methods</a>
+      </li>
+    </ul>
+  </li>
+  <li class="dropdown profile ">
+    <a class="dropdown-toggle"  id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
@jaypal412 again `dropdownMenuButton`, like I said last time we have to keep consistent names.

Reply:
Done